New page: SVGElement.autofocus (#38809)
mdn/content: files/en-us/web/api/svgelement/autofocus/index.md

---
title: "SVGElement: autofocus property"
short-title: autofocus
slug: Web/API/SVGElement/autofocus
page-type: web-api-instance-property
browser-compat: api.SVGElement.autofocus
---

{{APIRef("SVG")}}

> The **`autofocus`** property of the {{domxref("SVGElement")}} interface contains a boolean value reflecting the [`autofocus`](/en-US/docs/Web/HTML/Global_attributes/autofocus) HTML global attribute. It indicates whether the SVG element should be focused when the page loads or when the element becomes shown, if the SVG element is inside a {{htmlelement("dialog")}} or a [popover](/en-US/docs/Web/HTML/Global_attributes/popover).

Author: ```suggestion
The **`autofocus`** property of the {{domxref("SVGElement")}} interface contains a boolean value reflecting the [`autofocus`](/en-US/docs/Web/HTML/Global_attributes/autofocus) HTML global attribute. It indicates whether the SVG element should be focused when the page loads or when the element becomes shown if the SVG element is inside a {{htmlelement("dialog")}} or a [popover](/en-US/docs/Web/HTML/Global_attributes/popover).
```

> The **`autofocus`** property of the {{domxref("SVGElement")}} interface represents a boolean value reflecting the [`autofocus`](/en-US/docs/Web/HTML/Global_attributes/autofocus) HTML global attribute, which indicates whether the SVG element should be focused when the page loads or when the element become shown if specified in an element inside HTML{{htmlelement("dialog")}} elements or elements whose [`popover`](/en-US/docs/Web/HTML/Global_attributes/popover) attribute is set.

Reviewer: ```suggestion
The **`autofocus`** property of the {{domxref("SVGElement")}} interface contains a boolean value reflecting the [`autofocus`](/en-US/docs/Web/HTML/Global_attributes/autofocus) HTML global attribute. It indicates whether the SVG element should be focused when the page loads or when the element becomes shown, if the SVG element is inside a {{htmlelement("dialog")}} or a [popover](/en-US/docs/Web/HTML/Global_attributes/popover).
```
